nixos/modules/security/acme/default.nix

```diff
@@ -274,10 +315,15 @@ let
         # Default permissions make the files unreadable by group + anon
         # Need to be readable by group
         chmod 640 *
+        ${optionalString (!isNull(lockfileName)) ''
+          # wrap up subshell, assign lock file to descriptor 3
+          ) 3> /var/lib/acme/locks/${lockfileName}
```

Comment:
Locks are generally better suited in a partition like /run/ where we get rid of old files on a reboot. This is not as important on posix locks, but still.

Reply:
Moved lock files to `/run/`, introduced systemd-tmpfiles for directory and permission management again, and extracted lock file management into separate service.